drivers/overlay/check_115.go

```diff
@@ -1,8 +1,10 @@
+//go:build !go1.16
 // +build !go1.16
 
 package overlay
 
 import (
+	"io/fs"
```

Comment:
This source file is a fallback for the sake of compilers earlier than 1.16, which don't have `io/fs` in their standard library.  You want to remove this file and merge `check_116.go` into `check.go` instead.